internal/api/grpc_service.go

```diff
@@ -7,7 +7,6 @@ import (
 
 	"github.com/DIMO-Network/device-definitions-api/internal/core/commands"
 	"github.com/DIMO-Network/device-definitions-api/internal/core/models"
-	coremodels "github.com/DIMO-Network/device-definitions-api/internal/core/models"
```

Comment:
This seems like a good change. I wonder if the renaming was meant to avoid confusion with the SQLBoiler models.

Reply:
The problem is models is still used predominantly here, coremodels was used by just two functions I think. I can make a new branch to re-introduce coremodels because changing all of them in this PR will introduce a lot of irrelevant changes

Reply:
I think it’s perfectly fine the way you have it.